Source/AssetRipper.Processing/AnimatorControllers/VirtualAnimationFactory.cs

```diff
@@ -424,10 +431,11 @@ private static IAnimatorState CreateAnimatorState(ProcessedAssetCollection virtu
 			animatorStateTransition.Name = TOS[Transition.UserID];
 			animatorStateTransition.IsExit_C1101 = Transition.IsExit();
 
+			animatorStateTransition.Atomic_C1101 = Transition.Atomic;
 			animatorStateTransition.TransitionDuration_C1101 = Transition.TransitionDuration;
 			animatorStateTransition.TransitionOffset_C1101 = Transition.TransitionOffset;
-			animatorStateTransition.ExitTime_C1101 = Transition.GetExitTime();
-			animatorStateTransition.HasExitTime_C1101 = Transition.GetHasExitTime();
+			animatorStateTransition.ExitTime_C1101 = Transition.ExitTime;
+			animatorStateTransition.HasExitTime_C1101 = Transition.HasExitTime;
```

Comment:
Why is this being changed?

Reply:
If the `IAnimatorController` and `IAnimatorStateTransition` have the same unity version (which they normally would), this should have no effect, which means you didn't need to change it. Please change it back so that the code can handle a unity version disparity.